test/python/oracle_compiler/test_oracle.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Tests oracle as a gate."""
+
+from qiskit.test import QiskitTestCase
+
+from qiskit.circuit.oracle.compile_oracle import compile_oracle
```

Comment:
I should just be able to do `from qiskit.circuit.oracle import compile_oracle`, can you edit the init?

Reply:
Just checking. `compile_oracle` is imported `as oracle` so the decorator is `@oracle`. Do you want both?
```
from .compile_oracle import compile_oracle as oracle
from .compile_oracle import compile_oracle
```

Alternative, I can use `from qiskit.circuit.oracle import oracle as compile_oracle` in the test. Same to me.